src/all/toomics/src/eu/kanade/tachiyomi/extension/all/toomics/ToomicsGlobal.kt

```diff
@@ -155,6 +185,40 @@ abstract class ToomicsGlobal(
         return GET(page.imageUrl!!, newHeaders)
     }
 
+    // ================================== Interceptors =================================
+
+    private fun ageVerificationWarning(chain: Interceptor.Chain): Response {
+        val request = chain.request()
+        val response = chain.proceed(request)
+        if (response.request.url.pathSegments.any { it.equals("age_verification", true) }) {
+            throw IOException("Use WebView for age verification")
```

Comment:
What is the age verification? Cookies? Js challenge? Should automate it if it's not a captcha / login thing.